tvtk/tests/test_visual.py

```diff
@@ -167,6 +170,8 @@ def test_box(self):
         # Then
         x = np.sqrt(2)
         bounds = get_bounds((0.,0,0), (x, x, 1.0))
+        if vtk_major_version >= 9:  # XXX probably a bug somewhere
+            bounds = [-0.5,  0.5, -0.5,  0.5, -0.5,  0.5]
```

Comment:
This and the one below are the two changes I'm least happy with. No idea why this changed, but it's at least reproducible across systems.

Reply:
This might be an issue with the Box object.  I don't think this is correct.  I will merge this for now and investigate this in a subsequent PR.

Reply:
Can you also remove this change as this is definitely a bug in box.